lib/zendesk_apps_tools/common.rb

```diff
@@ -10,6 +41,10 @@ def say_error(msg)
     end
 
     def get_value_from_stdin(prompt, opts = {})
+      if options[:unattended]
```

Comment:
not sure, in the context of translations what is input? the app code? yml or json files? the remote API endpoints we use?